meta/etcd.go

```diff
@@ -0,0 +1,100 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package meta
+
+import (
+	"context"
+	"time"
+
+	"cqfn.org/degitx/discovery"
+	"cqfn.org/degitx/misc"
+
+	ma "github.com/multiformats/go-multiaddr"
+	"go.etcd.io/etcd/clientv3"
+)
+
+type etcd struct {
+	Endpoints []string
+}
+
+func NewEtcd(endpoints ...string) (Storage, error) {
+	var urls []string
+	for _, endpoint := range endpoints {
+		multiaddr, err := ma.NewMultiaddr(endpoint)
+		if err != nil {
+			return nil, err
+		}
+		addr := new(discovery.MaNetworkAddr)
```

Comment:
@stepanvalyavskiy I think it's better to require address type explicitly, we have two options here:
 - reqiure `endpoints ...Multiaddr`
 - require `endpoints ...string` - as plain host names

I think it's OK to use plain host strings here, since etcd client supports it, not multiaddr